python/data_extraction/export_data.py

```diff
@@ -12,6 +13,11 @@
 
 config_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'config', 'data_export_fields.json')
 
+document_types_excluded_from_the_topic_taxonomy_filename = \
```

Comment:
The code to load in a list of blacklisted document might be better living in its own class

Reply:
Yeah, that's sensible, I've extracted it to the data module in this commit e2a583a .